LayoutTests/platform/wpe/accessibility/roles-exposed-expected.txt

```diff
@@ -322,6 +322,12 @@ mtr
 mtd
       AXRole: AXCell
 
+menu
+      AXRole: AXList
+
+li
+      AXRole: AXListItem
+
```

Comment:
The order here is correct! But why didn't you update `LayoutTests/platform/gtk/accessibility/roles-exposed-expected.txt` too?

https://searchfox.org/wubkat/rev/8e1bf48c90b9cc1784acfc301f5cc79d25d622f4/LayoutTests/platform/gtk/accessibility/roles-exposed-expected.txt#324